src/js/serial_backend.js

```diff
@@ -114,7 +114,7 @@ function connectDisconnect() {
         const selected_baud = PortHandler.portPicker.selectedBauds;
         const selectedPort = portName;
 
-        if (selectedPort === 'DFU') {
+        if (['DFU', 'noselection'].includes(selectedPort)) {
             $('select#baud').hide();
             return;
         }
```

Comment:
Hide or not the baud input must not be done here with jquery. It's being done at the Vue component. This part of code probably must be removed and fixed there if it needs to.

Reply:
This is the condition to show/hide the baud:
https://github.com/betaflight/betaflight-configurator/blob/617790b239cd6743bd7731a3d5b46a18fb6a9b21/src/components/port-picker/PortsInput.vue#L62-L65

Reply:
Did not add that line :) Probably should have done:

```
if (selectedPort === 'noselection')) {
    return;
}
```